gdk/aws_client_utils/S3ClientUtils.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from botocore.exceptions import ClientError
+
+
+class S3ClientUtils:
+    """
+    S3 client utils wrapper
+    """
+
+    def __init__(self, project_configuration, service_clients):
+        self.project_config = project_configuration
+        self.s3_client = service_clients["s3_client"]
+
+    def create_bucket(self, bucket, region):
```

Comment:
```suggestion
    def create_bucket(self, bucket, region = "us-east-1"):
```

Reply:
S3 boto3 client in us-east-1 throws `InvalidLocationConstraint` when used with `CreateBucketConfiguration` parameter. `CreateBucketConfiguration` can't be set to either `None` or `{}`.